src/config.c

```diff
@@ -2165,6 +2159,14 @@ static int numericParseString(typeData data, sds value, const char **err, long l
             return 1;
     }
 
+    /* Attempt to parse as an octal number */
+    if (data.numeric.flags & OCTAL_CONFIG) {
+        char* eptr;
+        *res = strtoll(value, &eptr, 8);
+        if (*eptr == '\0')
+            return 1;
```

Comment:
A comment may help clarify how we detect non-octal digits.

Do we also need to check that we didn't get an empty string by checking eptr == value?

```suggestion
        *res = strtoll(value, &eptr, 8);
        if (*eptr == '\0')
            return 1; /* some non-octal digits remain */
        if (eptr == value)
            return 1; /* no octal digits found */
```

Reply:
I don't follow why we would do the second check? The man pages say we should be checking for errno here instead of the returned value, but it looks like within this page we don't consistently check it anyways.

Reply:
..and in util memtoull(), observable with
`./src/redis-server --maxmemory-clients ''`